src/Textures.cpp

```diff
@@ -1015,7 +1015,7 @@ void TextureCache::_loadBackground(CachedTexture *pTexture)
 		params.width = pTexture->realWidth;
 		params.height = pTexture->realHeight;
 		params.format = colorFormat::RGBA;
-		params.internalFormat = glInternalFormat;
+		params.internalFormat = gfxContext.convertGHQTextureFormat(u32(glInternalFormat));
```

Comment:
Ok, it seems like the extension OES_rgb8_rgba8 is allowing the additional color formats, but it's still not fully compatible with GLES 3.0.